resource_obmcs_core_instance_test.go

```diff
@@ -4,88 +4,37 @@ package main
 
 import (
 	"testing"
-	"time"
 
 	"github.com/MustWin/baremetal-sdk-go"
 	"github.com/hashicorp/terraform/helper/resource"
-	"github.com/hashicorp/terraform/helper/schema"
-	"github.com/hashicorp/terraform/terraform"
-
-	"github.com/stretchr/testify/suite"
-
-	"github.com/oracle/terraform-provider-baremetal/crud"
 )
 
-type ResourceCoreInstanceTestSuite struct {
-	suite.Suite
```

Comment:
On the fence about removing the Suite--at minimum it's consistent, especially if tests need to be expanded for a particular resource.